lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -217,6 +219,25 @@ def compile_props(constant)
           end.join("\n")
         end
 
+        sig { params(constant: Module).returns(String) }
+        def compile_enums(constant)
+          return "" unless T::Enum > constant
```

Comment:
Small nitpick, but can we swap the order of this comparison please? I find it much easier to read `constant < T::Enum` as "`constant` inherits from `T::Enum`" personally.

```suggestion
          return "" unless constant < T::Enum
```

Reply:
It turns out that this advice was pretty bad, the `XPath` constant from the `capybara` gem messes with the `<` operator 🤦 :
```
pry> XPath < T::Enum
=> #<XPath::Expression:0x0000559c53120378
 @arguments=
  [:<, #<XPath::Expression:0x0000559c531203c8 @arguments=[], @expression=:this_node>, T::Enum],
 @expression=:binary_operator>
pry> T::Enum > XPath
=> nil
pry> ancestors_of(XPath)
=> [XPath, XPath::DSL]
```

I'll revert the comparison order back again to fix this and will do the same for other locations that might be doing similar checks too.